scripts/package.js

```diff
@@ -6,7 +6,7 @@ const os = require('os');
 const zlib = require('zlib');
 const embed = require('./embed');
 
-const nodeVersion = '9.2.0';
+const nodeVersion = process.argv.slice(2)[0];
```

Comment:
let's do `slice(0)[2]` for stylistic reasons

Reply:
cool will do